packages/app-store/plausible/components/InstallAppButton.tsx

```diff
@@ -1,18 +0,0 @@
-import type { InstallAppButtonProps } from "@calcom/app-store/types";
```

Comment:
One should not add InstallAppButton component separately now. It's automatically handled by App Store. 
Adding it actually disables OmniInstallation behaviour.